features/theme.feature

```diff
@@ -3,7 +3,7 @@ Feature: Manage WordPress themes
   Scenario: Installing and deleting theme
     Given a WP install
 
-    When I run `wp theme install p2`
+    When I run `wp theme install twentytwelve`
     Then STDOUT should not be empty
 
     When I run `wp theme status p2`
```

Comment:
The following steps all need to be updated as well because we're no longer installing p2